images/demo/jupyter/kamu.py

```diff
@@ -28,15 +28,15 @@ def resolve_dataset_ref(dataset_ref):
     else:
         # Multi-tenant
         # Assuming layout <account_name>/<dataset_id>/info/alias
-        account_path, _ = dataset_ref.split("/", 1)
+        account_path, name = dataset_ref.split("/", 1)
         if os.path.isdir(account_path):
             for dataset_id in os.listdir(account_path):
                 alias_path = os.path.join(account_path, dataset_id, "info", "alias")
                 if not os.path.exists(alias_path):
                     continue
                 with open(alias_path) as f:
                     alias = f.read().strip()
-                if alias != dataset_ref:
+                if alias != name:
```

Comment:
This was a bug visible only when using `kamu notebook` with multi-tenant local repos.